src/media/css/account.styl

```diff
@@ -81,6 +81,10 @@
         display: none;
     }
 }
+.html-rtl .region span {
```

Comment:
You could make a similar argument that says "I know that when editing the styles for <`.region span`>, they are all in one place, rather than scattered across 5 files.